docs/operating-procedures.md

```diff
@@ -20,42 +20,48 @@ This should only be used for public web access, otherwise private tokens or view
 This means if the back-end OMERO data is modified the cache may become stale.
 A special cache-buster port `9000` is configured on `idr-proxy`.
 Any requests to this port will force a cache refresh.
+
+For example:
+
+    ssh idr-proxy -L 9000:localhost:9000
+
+And open http://localhost:9000.
+Due to the way OMERO.web redirects you may be redirected to a host without the port, in which case add the port back.
+
 In the case of large screens this may lead to a significant slowdown of the server, though it does also show the importance of the cache.
 
 You can force a complete refresh of the cache by removing everything under `/var/cache/nginx/`.
 
+**WARNING**: Never log in to OMERO.web on any idr-proxy port to prevent incorrect state being cached.
```

Comment:
We might should even disable the `/login` path, then.

Reply:
https://github.com/IDR/deployment/pull/112